.changes/v3.0.0/557-removals.md

```diff
@@ -0,0 +1 @@
+* Removed deprecated resources `vcd_dnat` `vcd_snat` `vcd_firewall_rules` [GH-557]
```

Comment:
1. Polishing:
```suggestion
* Removed deprecated resources `vcd_dnat`, `vcd_snat` and `vcd_firewall_rules` [GH-557]
```

2. What about adding all the removed resources in 3.0 to a new section in the index page, so it's clearly visible? CC: @vbauzysvmware @Didainius

Reply:
Section added